### PR TITLE
Troubleshooting catalog docs

### PR DESCRIPTION
**Pull request by iain-b**

### CODE REVIEW

File: content/docs/details/troubleshooting-the-catalog/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: Troubleshooting the Catalog
+publishedDate: '2022-05-17T21:00:00.0Z'
+description: How to resolve issues getting data into the catalog
+---
+
+## About Locations
+
+The backstage catalog uses location entities to point to external sources (in version control) which defined entities.
```
Comment (Xantier) [inline content/docs/details/troubleshooting-the-catalog/index.md:9]:
```suggestion
The backstage catalog uses location entities to point to external sources which define entities.
```
Not necessarily SCM, we have S3 enabled as well right?

**Review by Xantier — APPROVED** [review]

File: content/docs/details/troubleshooting-the-catalog/index.md (hunk 1 above)
Comment (dtuite) [inline content/docs/details/troubleshooting-the-catalog/index.md:9]: Could we go into more depth here perhaps? What does a location entity look like? Like I've been using B backstage for 2 years and I'm honestly still confused about locations and how they get interpreted by Backstage.

Comment (Xantier) [reply content/docs/details/troubleshooting-the-catalog/index.md]: I'll continue on top of these docs and add more stuff regarding location mgmt etc.

**Comment by Xantier** [comment]
Merging in Iain's absence

**Pull request merged**